gp-includes/routes/translation.php

```diff
@@ -593,4 +593,37 @@ private function can_approve_translation_or_forbidden( $translation ) {
 		}
 		$this->can_or_forbidden( 'approve', 'translation-set', $translation->translation_set_id );
 	}
+
+	/**
+	 * Get the glossary for the translation set
+	 *
+	 * This will also fetch contents from a potential master glossary, as well as from a parent project.
+	 *
+	 * @param  GP_Translation_Set $translation_set translation_set for which to retrieve the glossary.
+	 * @param  GP_Project         $project         project for finding potential parent projects.
+	 * @return GP_Glossary                         extended glossary
+	 */
+	protected function get_extended_glossary( $translation_set, $project ) {
```

Comment:
* Summary should be in third-person singular, see https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#documenting-tips
* Needs a `@since` tag.
* The `@return` line should not be aligned with the `@param` lines.
* Descriptions for params should start with a capital letter.